### ISSUE TITLE
Update from upstream?

### ISSUE THREAD

**Issue by maxime1992** [jossef/material-design-icons-iconfont#15]
Thanks for the great work :)

Would you mind to update from upstream? Some icons have been added and are now missing on this fork.

Thanks :raised_hands:

**Comment by jossef** [comment]
@maxime1992 
are these icons included in the latest release on the official [releases page](https://github.com/google/material-design-icons/releases)?

**Comment by maxime1992** [comment]
@jossef I should have written the missing ones. If I remember well I was looking for `power_off` which is available here: https://material.io/tools/icons/?search=power_off&style=baseline

And I believe this app is based on the repo you linked, but I'm not 100% sure.

**Comment by bentaly** [comment]
drag_indicator seems to be missing from 3.0.3

**Comment by jossef** [comment]
TL;DR - I've verified that this is a bug. 

### We are synced with the 'official' release
Important to mention, this repo's master branch is *up-to-date* with the official iconfont release [v3.0.1](https://github.com/google/material-design-icons/releases/tag/3.0.1)

Which means the **problem originate from the source!**. 

### This is undoubtedly a serious problem

see ([issue 864](https://github.com/google/material-design-icons/issues/864), [issue 860](https://github.com/google/material-design-icons/issues/860) and the list goes on ...). 

My guess is that Google abandoned the maintenance of their iconfont [GitHub repository](https://github.com/google/material-design-icons).


## Solution?

To address this issue, after some self-brainstorming - I've done some research and it looks that the iconfont is maintained and published as of today into their fonts CDN - [Google Fonts](https://fonts.google.com/) 

    https://fonts.googleapis.com/icon?family=Material+Icons

I'll scrape the most updated font files from their CDN and make a new release ASAP

----

Sorry for the late response, I'll push a new release later today

**Comment by jossef** [comment]
closing, new release + demo page
missing icons are working in 4.0.1
https://jossef.github.io/material-design-icons-iconfont/

![image](https://user-images.githubusercontent.com/1287098/47607612-cf86ce00-da2a-11e8-809f-e5dabd6497b4.png)
